File: sample-kanban-jvm/src/main/kotlin/sample/testcontainer/kanban/models/Message.kt
package sample.testcontainer.kanban.models

import jakarta.persistence.*
import java.time.LocalDateTime

@Entity
@Table(name = "message", schema = "kanban")
data class Message(
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    var id: Long? = null,
    var content: String? = null,
    @Column(updatable = false)
    @Temporal(TemporalType.TIMESTAMP)
    var created: LocalDateTime? = null,
    @ManyToOne(optional = false)
    var person: Person? = null,
    @ManyToOne(optional = false)
    var task: Task? = null,
)

File: sample-kanban-go/app/models/message.go
package models

import "time"

type Message struct {
	Id       int64      `db:"id" goqu:"skipinsert"`
	PersonId int64      `db:"person_id"`
	TaskId   int64      `db:"task_id"`
	Content  string     `db:"content"`
	Created  *time.Time `db:"created" goqu:"omitnil,skipupdate"`
}
